cmd/monitor/app/options.go

```diff
@@ -4,17 +4,19 @@ import (
 	"io/ioutil"
 	"time"
 
+	comp_v1 "github.com/atlassian/voyager/pkg/apis/composition/v1"
 	creator_v1 "github.com/atlassian/voyager/pkg/apis/creator/v1"
 	"github.com/atlassian/voyager/pkg/options"
 	"github.com/pkg/errors"
 	"sigs.k8s.io/yaml"
 )
 
 type Options struct {
-	ServiceDescriptorName  string                 `json:"name"`
-	Location               options.Location       `json:"location"`
-	ExpectedProcessingTime time.Duration          `json:"expected_processing_time"`
-	ServiceSpec            creator_v1.ServiceSpec `json:"service_spec"`
+	ServiceDescriptorName  string                     `json:"name"`
+	ServiceDescriptor      *comp_v1.ServiceDescriptor `json:"service_descriptor"`
```

Comment:
Ideally we should use camel case everywhere but I see that is already not the case :( Would be great to change that in a separate PR.